<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSMmainwindowClass</class>
 <widget class="QMainWindow" name="DSMmainwindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1657</width>
    <height>1041</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gdismainwindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="TreeWidget" name="treeWidget">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>245</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>200</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>Images</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>91</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Images:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>80</y>
          <width>68</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>80</y>
          <width>68</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>+</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>110</y>
          <width>45</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>110</y>
          <width>45</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>140</y>
          <width>45</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_4">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>140</y>
          <width>45</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_5">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>170</y>
          <width>45</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_6">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>170</y>
          <width>45</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>110</y>
          <width>68</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>v1:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>140</y>
          <width>68</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>v2:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>170</y>
          <width>68</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>v3:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Options</string>
       </attribute>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>111</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Hide Cells</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>111</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Hide Cells</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Hide Cells</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="2" rowspan="4">
     <widget class="GLWidget" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1280</width>
        <height>720</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="TableWidget" name="tableWidget">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>280</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>242</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string>Name</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Symbol</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Cova</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>X</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Y</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Z</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Selected Atom</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>/</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>/</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>/</string>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>/</string>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>/</string>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string>/</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1657</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen_Structure">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave_Structure"/>
    <addaction name="actionSave_Structure_As"/>
    <addaction name="actionClose_Structure"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCreate_new_Structure"/>
    <addaction name="actionEdit_Structure"/>
   </widget>
   <addaction name="menuOpen_Structure"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Structure"/>
  </widget>
  <action name="actionOpen_Structure">
   <property name="icon">
    <iconset>
     <normaloff>src/icons/folder.xpm</normaloff>src/icons/folder.xpm</iconset>
   </property>
   <property name="text">
    <string>Open Structure</string>
   </property>
   <property name="toolTip">
    <string>Open Structure</string>
   </property>
  </action>
  <action name="actionCreate_new_Structure">
   <property name="text">
    <string>Create new Structure</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionClose_Structure">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSave_Structure">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_Structure_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionEdit_Structure">
   <property name="text">
    <string>Edit Structure</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>../src/graphics/glwidget.h</header>
   <slots>
    <signal>mouseClicked(glm::vec4,glm::vec3,glm::mat4)</signal>
    <signal>atomMoved(glm::vec3)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>TreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>../src/widgets/treewidget.h</header>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>../src/widgets/tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="DSMmainwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DSMmainwindowClass</receiver>
   <slot>setNegV1ImageCounter(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>719</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DSMmainwindowClass</receiver>
   <slot>setPosV1ImageCounter(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>719</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DSMmainwindowClass</receiver>
   <slot>setNegV2ImageCounter(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>749</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DSMmainwindowClass</receiver>
   <slot>setPosV2ImageCounter(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>749</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DSMmainwindowClass</receiver>
   <slot>setNegV3ImageCounter(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>779</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DSMmainwindowClass</receiver>
   <slot>setPosV3ImageCounter(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>779</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openGLWidget</sender>
   <signal>mouseClicked(glm::vec4,glm::vec3,glm::mat4)</signal>
   <receiver>DSMmainwindowClass</receiver>
   <slot>mouseClicked(glm::vec4,glm::vec3,glm::mat4)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1003</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openGLWidget</sender>
   <signal>atomMoved(glm::vec3)</signal>
   <receiver>DSMmainwindowClass</receiver>
   <slot>atomMoved(glm::vec3)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1003</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mainToolBar</sender>
   <signal>actionTriggered(QAction*)</signal>
   <receiver>DSMmainwindowClass</receiver>
   <slot>openStructure()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>849</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>DSMmainwindowClass</receiver>
   <slot>atomInfoEdited(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>DSMmainwindowClass</receiver>
   <slot>treewidgetSelection(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menuBar</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>DSMmainwindowClass</receiver>
   <slot>menuActionTriggered(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>849</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openStructure()</slot>
  <slot>setPosV1ImageCounter(int)</slot>
  <slot>setNegV1ImageCounter(int)</slot>
  <slot>setPosV2ImageCounter(int)</slot>
  <slot>setNegV2ImageCounter(int)</slot>
  <slot>setPosV3ImageCounter(int)</slot>
  <slot>setNegV3ImageCounter(int)</slot>
  <slot>mouseClicked(glm::vec4,glm::vec3,glm::mat4)</slot>
  <slot>atomMoved(glm::vec3)</slot>
  <slot>atomInfoEdited(QTableWidgetItem*)</slot>
  <slot>treewidgetSelection(QTreeWidgetItem*)</slot>
  <slot>menuActionTriggered(QAction*)</slot>
 </slots>
</ui>
